### ISSUE TITLE
Merge exakat/php-static-analysis-tools

### ISSUE THREAD

**Issue by andygrunwald** [analysis-tools-dev/static-analysis#64]
I found https://github.com/exakat/php-static-analysis-tools

Maybe there are some tools that are not listed here and it might make sense to complete awesome-static-analysis with those even more :)

**Comment by danielleitcs** [comment]
To add to this, you might also find user reviews for many of the tools mentioned in these lists on IT Central Station to be helpful.

As an example, this user writes in his review of Veracode, "Veracode static analysis allows us to pinpoint issues - from a simple hard-coded test password, to more serious issues - and saves us lot of time. For example, it raises a flag about a problematic third-party DLL before development invests time heavy using it." You can read the rest of his review [here](https://www.itcentralstation.com/product_reviews/veracode-review-49429-by-informatab29/tzd/c574-sf-129).

**Comment by Harvester57** [comment]
Well, that's a nice spam for the website that employs you as a Social Media & Content Manager...

**Comment by andygrunwald** [comment]
@Harvester57 i don't have any relationship to @exakat or the php-static-analysis-tools listing.
Furthermore, it is a ticket. Not a "you have to"/force statement.

An alternative could be to just link the mentioned repo here. Any thoughts?

**Comment by mre** [comment]
@andygrunwald's suggestion to merge the lists is totally reasonable.
This is not spam, @Harvester57. The two projects have a different scope: while ours analysis tools for all languages, exacat's list is focused on PHP tools. Of course there is a bit of overlap.

@andygrunwald , we already have a link to the exacat list in the [more collections section](https://github.com/mre/awesome-static-analysis#more-collections), I would still love to integrate the missing tools here when I find the time, though. Any pull requests are highly appreciated. 👍

**Comment by Harvester57** [comment]
I guess you guys missed the part when @danielleitcs spammed this post and then removed the message, as show in the screen of the mail notification I got. That's the message I answered to, and it was not directed toward you @andygrunwald. Apologies for the misunderstanding.

![capture du 2018-09-28 11-45-18](https://user-images.githubusercontent.com/7060295/46201380-4b9ad280-c314-11e8-8741-e517d70a19cf.png)
